src/Microsoft.Extensions.FileSystemGlobbing/Sources/HashCodeCombiner.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+// TODO use the package when https://github.com/dotnet/sdk/issues/495
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Internal
+{
+    internal struct HashCodeCombiner
```

Comment:
This was added from aspnet/Common because we can't take a reference on Microsoft.Extensions.HashCodeCombiner.Sources. cref https://github.com/dotnet/sdk/issues/495